src/iOS/Xamarin.iOS/Samples/Layers/FeatureLayerDefinitionExpression/FeatureLayerDefinitionExpression.cs

```diff
@@ -40,93 +40,107 @@ public override void ViewDidLoad()
+            {
+            }
         }
 
         private async void Initialize()
         {
-            // Create new Map with basemap
-            Map myMap = new Map(Basemap.CreateTopographic());
+            // Create new Map with basemap.
+            Map map = new Map(Basemap.CreateTopographic());
 
-            // Create a mappoint the map should zoom to
+            // Create a point the map should zoom to.
             MapPoint mapPoint = new MapPoint(-13630484, 4545415, SpatialReferences.WebMercator);
 
-            // Set the initial viewpoint for map
-            myMap.InitialViewpoint = new Viewpoint(mapPoint, 90000);
+            // Set the initial viewpoint for map.
+            map.InitialViewpoint = new Viewpoint(mapPoint, 90000);
 
-            // Provide used Map to the MapView
-            _myMapView.Map = myMap;
+            // Provide used Map to the MapView.
+            _myMapView.Map = map;
 
-            // Create the uri for the feature service
+            // Create the URI for the feature service.
             Uri featureServiceUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/SF311/FeatureServer/0");
 
-            // Initialize feature table using a url to feature server url
+            // Initialize feature table using a URL to feature server.
             ServiceFeatureTable featureTable = new ServiceFeatureTable(featureServiceUri);
 
-            // Initialize a new feature layer based on the feature table
+            // Initialize a new feature layer based on the feature table.
             _featureLayer = new FeatureLayer(featureTable);
 
-            // Load the layer
+            // Load the layer.
             await _featureLayer.LoadAsync();
 
-            // Check for the load status. If the layer is loaded then add it to map
+            // Check for the load status. If the layer is loaded then add it to map.
             if (_featureLayer.LoadStatus == Esri.ArcGISRuntime.LoadStatus.Loaded)
             {
-                //Add the feature layer to the map
-                myMap.OperationalLayers.Add(_featureLayer);
+                // Add the feature layer to the map.
+                map.OperationalLayers.Add(_featureLayer);
             }
         }
 
         private void OnApplyExpressionClicked(object sender, EventArgs e)
         {
-            // Adding definition expression to show specific features only
+            // Adding definition expression to show specific features only.
             _featureLayer.DefinitionExpression = "req_Type = 'Tree Maintenance or Damage'";
         }
 
         private void OnResetButtonClicked(object sender, EventArgs e)
         {
-            // Reset the definition expression to see all features again
-            _featureLayer.DefinitionExpression = "";
+            // Reset the definition expression to see all features again.
+            _featureLayer.DefinitionExpression = String.Empty;
```

Comment:
There's been no benefit to using `string.Empty` over `""` since .NET 2.0. Use whichever one is more readable. IMO that's `""`.